engine/src/engine.rs

```diff
@@ -1208,6 +1184,7 @@ pub fn get_nonce<I: IO>(io: &I, address: &Address) -> U256 {
         .unwrap_or_else(|_| U256::zero())
 }
 
+#[cfg(test)]
```

Comment:
Do we actually use this function in tests or it can be deleted too?

Reply:
This function is used in tests now only. But I decided to leave it just in case.